<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>201</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="toolButton_Prev">
         <property name="text">
          <string>Prev</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/Prev</normaloff>:/images/Prev</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_Play">
         <property name="text">
          <string>Play</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/Play</normaloff>:/images/Play</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_Stop">
         <property name="text">
          <string>Stop</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/Stop</normaloff>:/images/Stop</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_Next">
         <property name="text">
          <string>Next</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/Next</normaloff>:/images/Next</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="sliderProgress">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
